projects/js-packages/ai-client/src/suggestions-event-source/index.ts

```diff
@@ -72,6 +72,9 @@ export default class SuggestionsEventSource extends EventTarget {
 	isPromptClear: boolean;
 	controller: AbortController;
 
+	// Flag to detect if the unclear prompt event was already dispatched
+	_error_unclear_prompt_triggered: boolean;
```

Comment:
JS code should use camelCase
```suggestion
	errorUnclearPromptTriggered: boolean;
```